perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -373,7 +383,7 @@ sub _in_cab_logs {
 
         my $logdate = DateTime::Format::Strptime->new( pattern => '%Y-%m-%dT%H:%M:%S' )->parse_datetime( $_->{LogDate} );
 
-        if ( $_->{Uprn} ) {
+        if ( $_->{Uprn} && $_->{Uprn} eq $property->{uprn} ) {
```

Comment:
I guess it is possible for the UPRN to not match because you are also now fetching the USRN logs above?

Reply:
Yep exactly. The USRN logs contain both street-level logs (which have a blank UPRN field) and any property logs for that USRN (hence the need to filter out irrelevant ones here).